app/controllers/api/subcollections/policies.rb

```diff
@@ -41,9 +41,6 @@ def parse_policy(data, collection, klass)
 
         guid = data["guid"]
         return klass.find_by(:guid => guid) if guid.present?
-
-        href = data["href"]
-        href =~ %r{^.*/#{collection}/#{CID_OR_ID_MATCHER}$} ? klass.find(from_cid(href.split('/').last)) : {}
```

Comment:
:point_up: https://github.com/ManageIQ/manageiq/pull/13671#issue-203521507 item number 3. Happy to be proved wrong, but I did some testing on this endpoint and couldn't get at it